src/elvis_code.erl

```diff
@@ -128,6 +129,16 @@ is_at_location(Node = #{attrs := #{location := {Line, NodeCol}}},
 is_at_location(_, _) ->
     false.
 
+-spec find_token(ktn_code:tree_node(), {integer(), integer()}) ->
+    undefined | {ok, map()}.
```

Comment:
This spec says it returns undefined, but it returns ``not_found`` and it's converted to undefined elvis_style.erl@609